time stamp
float64 time
int64 leftTicks
int64 rightTicks
float64 leftVel
float64 rightVel
